packages/amplify_core/lib/src/types/storage/copy_options.dart

```diff
@@ -12,11 +12,17 @@ class StorageCopyOptions
         AWSSerializable<Map<String, Object?>>,
         AWSDebuggable {
   /// {@macro amplify_core.storage.copy_options}
-  const StorageCopyOptions({this.pluginOptions});
+  const StorageCopyOptions({
+    this.pluginOptions,
+    this.buckets,
+  });
 
   /// plugin specific options for `Amplify.Storage.copy`.
   final StorageCopyPluginOptions? pluginOptions;
 
+  /// Optionally specify which bucket to target
+  final CopyBuckets? buckets;
+
   @override
   List<Object?> get props => [pluginOptions];
```

Comment:
```suggestion
  List<Object?> get props => [pluginOptions, buckets];
```

Reply:
Good catch, I accidentally deleted this when updating the implementation to use CopyBuckets instead of 2 StorageBucket.